pkg/rancher-desktop/utils/paths.ts

```diff
@@ -122,6 +122,26 @@ function getRdctlPath(): string | null {
   return rdctlPath;
 }
 
+function getLogDir() {
```

Comment:
If we're going to do this, we should remove the `script_logs` from `.gitignore`.

That being said, I'm not sure we want logs to go to the usual location in the situation this function in used (i.e. running as a script). Though I'm sure this will all change in the PR that comes after this. Just noting this in case it isn't what we want.

Reply:
In the next PR (which uses branch 5128-refactor-postinstall) the scripts should never import `paths.ts`

And in the past, when they did hit the version of `paths.ts` that hardwired all the paths in it, the scripts would have written to the usual log directories anyway (although I think they never did).

As an experiment, I did the following:
```
LOGDIR=$HOME/.local/share/rancher-desktop/.logs
rm $LOGDIR/*.log # (in the log dir)
git clean -ffdx pkg src scripts
npm run postinstall
ls -l $LOGDIR

.... settings.log rw---- 0 bytes
```

This is what I'd expect. After the PR for `5128-refactor-postinstall` $LOGDIR should be empty